assets/js/implicit-login.js

```diff
@@ -14,23 +17,44 @@ document.addEventListener('DOMContentLoaded', function() {
         return p;
     }, {});
 
-    var form = document.createElement('form');
-    form.setAttribute('method', 'post');
-    form.setAttribute('action', wpAuth0ImplicitGlobal.postUrl);
+    var form = document.createElement( 'form' );
+    form.setAttribute( 'method', 'post' );
+    form.setAttribute( 'action', wpAuth0ImplicitGlobal.postUrl );
 
-    var hiddenField = document.createElement('input');
-    hiddenField.setAttribute('type', 'hidden');
+    if ( data.hasOwnProperty( 'error' ) ) {
+        form.appendChild( createHiddenField( 'error', data.error ) );
+        form.appendChild( createHiddenField( 'error_description', data.error_description ) );
+    } else if ( data.hasOwnProperty( 'id_token' ) && data.hasOwnProperty( 'state' ) ) {
+        form.appendChild( createHiddenField( 'token', data.id_token ) );
+        form.appendChild( createHiddenField( 'state', data.state ) );
+    } else {
+        redirectWithoutQuery();
```

Comment:
Is this a valid path? When will it happen? Do you have an example? If you don't have one but still want to catch it, would be nice to log that query somewhere to debug and understand it I guess. If the authentication was initiated for "implicit flow" and the token is not there, there must be an error.

Reply:
It's valid as in it exists but without a hash we don't have much to do. The param triggers this JS to load and for the form to be excluded (param tells us that we need to process something from Auth0). If we don't have the information we need, then we need to cease processing, hence the redirect. 

If we have the Auth0 param but no `error` or no `state`, that could be an intermittent problem, in which case showing the login form to try again seems reasonable. I'm not sure I want to catch and log every possible outcome here, I think it's best to get someone back to a place where they can try again.